data_validation/cli_tools.py

```diff
@@ -791,7 +796,9 @@ def get_arg_list(arg_value, default_value=None):
             arg_value = str(arg_value)
         # arg_value = "hash_all"
         arg_list = json.loads(arg_value)
+        print(799)
```

Comment:
Are these print statements supposed to be here?

Reply:
removed the print(). Those were for debugging.